spec/javascripts/health-care/components/financial-assessment/AnnualIncomeSection.spec.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+
+import AnnualIncomeSection from '../../../../../_health-care/_js/components/financial-assessment/AnnualIncomeSection';
+
+describe('<AnnualIncomeSection>', () => {
+  const nullData = [{
+    veteranGrossIncome: null,
+    veteranNetIncome: null,
+    veteranOtherIncome: null,
+    spouseGrossIncome: null,
+    spouseNetIncome: null,
+    spouseOtherIncome: null,
+    childrenGrossIncome: null,
+    childrenNetIncome: null,
+    childrenOtherIncome: null
+  }];
+
+  it('Sanity check the component renders', () => {
+    const tree = SkinDeep.shallowRender(<AnnualIncomeSection
+        data={{ nullData }}
```

Comment:
The tests should pass without setting `nullData` and instead just passing an empty object: `data={{}}`. Since you're not testing anything about the data itself, no need to set it.